draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -607,7 +607,8 @@ as follows (line break added for readability):
 This is called the Combined Format for Presentation.
 
 The Holder MAY send any subset of the Disclosures to the Verifier, i.e.,
-none, multiple, or all Disclosures.
+none, multiple, or all Disclosures. The Holder MUST NOT send Disclosures
+or salt values for data that should not be revealed to the Verifier.
```

Comment:
salt is part of the disclosures, so maybe something like:
```suggestion
none, multiple, or all Disclosures. The Holder MUST NOT send Disclosures,
especiallt salt values, for data that should not be revealed to the Verifier.
```

Reply:
"especially" makes it sound like the salt is somehow more sensitive than the claim name/value. Which is not the case at all. The salt only adds randomness to the hash input  to make it infeasible to "reverse" the hash value by enumerating potential claim name/values into the hash function. The salt does nothing from the privacy perspective, if the claim is reveled.

Reply:
maybe something like this, which acknowledges that salt is part of the disclosure but doesn't over emphasize it: 
```suggestion
none, multiple, or all Disclosures. The Holder MUST NOT send Disclosures
(including salt values) for data that should not be revealed to the Verifier.
```
?

Reply:
I wanted to say something like "don't reveal the disclosures, but also don't take the salt values out of them and reveal them separately". 

Also I just notice that I don't like the 'should' in the original sentence.

```suggestion
none, multiple, or all Disclosures. For data that the Holder does not want to reveal
to the Verifier, the Holder MUST NOT send Disclosures or reveal the salt values in any
other way.
```

Reply:
I can live with that.

Reply:
me too